12주차/(P)17686/17686_cpp_이지영.cpp

```diff
@@ -0,0 +1,88 @@
+#include <string>
+#include <vector>
+#include <iostream>
+#include <algorithm>
+#include <cctype>
+
+using namespace std;
+
+
+//signal: aborted (core dumped) 에러로 채점 불가능
+
+bool sorttt(vector<string> a, vector<string> b) {
+    int aa = stoi(a[0]);
+    int bb = stoi(b[0]);
+    int aaa = tolower(aa);
+    int bbb = tolower(bb);
+    if (aaa < bbb) {
+        return true;
+    } else if (aaa > bbb) {
+        return false;
+    } else {
+        if (stoi(a[1]) <= stoi(b[1])) {
+            return true;
+        } else {
+            return false;
+        }
+    }

```

Comment:
이 식을 유지한다면 아래와 같이 리팩터링 가능합니다. (굳이 꼭 true, false를 리턴할 필욘 없어요)
```suggestion
    if (aaa == bbb) {
        return stoi(a[1]) <= stoi(b[1]);
    } 
    return aaa<bbb;
```

Reply:
리팩토링도 보여주시다니.. 정말 감사합니다! 👍